src/WebJobs.Extensions.DurableTask/ContextImplementations/RemoteOrchestratorContext.cs

```diff
@@ -71,15 +71,15 @@ internal bool TryGetOrchestrationErrorDetails(out Exception? failure)
             return hasError;
         }
 
-        internal void SetResult(IEnumerable<OrchestratorAction> actions, string customStatus)
+        internal void TrySetResult(IEnumerable<OrchestratorAction> actions, string customStatus)
```

Comment:
Non-blocking nit: Any method named `TryXXX` is usually expected to return a `bool` value indicating whether it succeeded or failed.

Reply:
yes, you're right. II chose `Try` to signal somehow that the method may throw, but I know the convention makes this confusing. I'll revert this change.

Reply:
fixed here: https://github.com/Azure/azure-functions-durable-extension/pull/2922/commits/090f8e729a0c584af3863209154f4ecd9ecd4c9c